client/src/client.rs

```diff
@@ -1,28 +1,24 @@
-use base64::prelude::BASE64_STANDARD;
-use base64::Engine;
-use core::str;
+use base64::{prelude::BASE64_STANDARD, Engine as _};
 use libsignal_core::{Aci, Pni};
-use std::default;
 use std::error::Error;
-use std::fmt::{self, format, Debug, Display};
 use surf::StatusCode;
 
 use common::web_api::{
     AccountAttributes, DeviceCapabilities, RegistrationRequest, RegistrationResponse,
-    UploadSignedPreKey,
 };
 use libsignal_protocol::{
-    IdentityKey, IdentityKeyPair, InMemSignalProtocolStore, KeyPair, KyberPreKeyRecord, PublicKey,
+    IdentityKey, IdentityKeyPair, InMemSignalProtocolStore, KeyPair, KyberPreKeyRecord,
     SignedPreKeyRecord,
 };
-use rand::rngs::OsRng;
-use rand::Rng;
-
-use crate::contact_manager::ContactManager;
-use crate::errors::{LoginError, RegistrationError};
-use crate::key_management::key_manager::{InMemoryKeyManager, KeyManager};
-use crate::server::{Server, ServerAPI};
-use crate::storage::{self, DeviceStorage, Storage};
+use rand::{rngs::OsRng, Rng};
+
+use crate::{
```

Comment:
Needs the websocket from the client to be included for the refactor to be relevant